openedx/core/djangoapps/user_authn/tests/utils.py

```diff
@@ -143,7 +143,7 @@ def test_anonymous_user(self):
 
     @ddt.data(*JWT_AUTH_TYPES)
     def test_self_user(self, auth_type):
-        resp = self.get_response(auth_type)
+        resp = self.get_response(auth_type, requesting_user=self.global_staff)
```

Comment:
I believe these are generic test cases for multiple endpoints not just this one, we shouldn't change these test cases and instead just remove this mixin's inheritance from `CertificatesListRestApiTest` because after restricting this endpoint to just staff users I believe it's not needed anymore.